custodian/vasp/handlers.py

```diff
@@ -439,7 +439,10 @@ def correct(self):
                                 break
                             except (IndexError, ValueError):
                                 pass
-            actions.append({"dict": "INCAR", "action": {"_set": {"NBANDS": int(1.1 * nbands)}}})
+            new_nbands = int(1.1 * nbands)
+            if new_nbands == nbands:
+                new_nbands += 1
```

Comment:
I think a better way to write this is `new_bands = max(int(1.1*nbands), nbands+1)`? It is one line instead of three.

Reply:
yep, that is even better